src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/ConfigurationBindingGeneratorTests.Baselines.cs

```diff
@@ -44,7 +43,6 @@ public class MyClass2 { }
 
         [Theory]
         [InlineData(LanguageVersion.Preview)]
```

Comment:
>LanguageVersion [](http://example.com/codeflow?start=20&length=15)

I assume we have a code checks language version to be at least 11 and produce diagnostics if it is not. If so, do we have a test for this?

Reply:
never mind, I see the test in other file now.